spec/index.md

```diff
@@ -25,6 +25,11 @@ This use case exports all resources needed for the US Core Data for Interoperabi
 
 This use case exports all resources needed to convey a patient's healththcare financial history, including Patient, ExplanationOfBenefit, Coverage, and Claim. While FHIR profiles are still being developed and standardized, see [https://bluebutton.cms.gov/developers/#core-resources](https://bluebutton.cms.gov/developers/#core-resources) for a full-fledged example.
 
+### Additional Use cases
+* Terminology data  e.g. to allow export of all ValueSet from a terminology server.
+* Provider data to export information about an system's full Practitioner, Location, and Organization list
+* Use in training machine learning algorithms on historical data
```

Comment:
Can you point me toward the GForge resolution where this came up? The other use cases are describing server functionality rather than what clients will do with data downstream. See this content in GF$21657.

Reply:
The provider use case is added based on the comments you (https://github.com/HL7/bulk-data/pull/3#discussion_r295600205) and other made. The ML use case I added as another, but it wasn't requested by anyone during the balloting comments. I believe it's a valid use case but we can remove it if you disagree or if that's not following protocol

Reply:
There are many, many valid use cases -- but we need to implement the specific changes that were requested in the ballot and adjudicated/voted on by the FHIR Infrastructure Workgroup. We are not at liberty to craft new language without proper review.

Reply:
There are many, many valid use cases -- but we need to implement the specific changes that were requested in the ballot and adjudicated/voted on by the FHIR Infrastructure Workgroup. We are not at liberty to craft new language without proper review.

Reply:
Yup got it. Ive removed that ML example